src/pages/register-a-beacon/beacon-use.tsx

```diff
@@ -63,42 +64,45 @@ const BeaconUse: FunctionComponent<DraftBeaconUsePageProps> = ({
       pageHeading={pageHeading}
       showCookieBanner={showCookieBanner}
       pageText={pageText}
-      headingType="legend"
     >
-      <FormGroup errorMessages={form.fields.environment.errorMessages}>
-        <RadioList conditional={true}>
-          <RadioListItem
-            id="maritime"
-            name={environmentFieldName}
-            label="Maritime"
-            hintText="This might include commercial or pleasure sailing / motor vessels or unpowered craft. It could also include sea-based windfarms and rigs/platforms."
-            value={Environment.MARITIME}
-            defaultChecked={
-              form.fields.environment.value === Environment.MARITIME
-            }
-          />
+      <FormFieldsetAndLegend pageHeading={pageHeading}>
```

Comment:
Reminder to put the page text from the `BeaconsForm` into this fieldset and legend heading